assignment4/promiseDemo.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Title</title>
+        <script>
+            let promise = new Promise(function(resolve, reject) {
+                if (delay(1000)) {
+                    resolve();
+                } else {
+                    reject();
+                }
+            });
+
+            promise.
+            then(function() {
+                console.log('Delay done');
+            }).catch(function() {
+                console.log('Delay not done');
+            })
+
+            function delay(ML) {
+                setTimeout(() => {} , ML);
+                return true;
+            }
```

Comment:
The assignment is incorrect. Create a delay function that will accept the delay seconds and then will return the promise which will resolve in given delay seconds.